book/src/configuration.md

```diff
@@ -67,13 +67,27 @@ left = ["mode", "spinner"]
 center = ["file-name"]
 right = ["diagnostics", "selections", "position", "file-encoding", "file-line-ending", "file-type"]
 separator = "│"
+mode.normal= "NORMAL"
+mode.insert= "INSERT"
+mode.select= "SELECT"
```

Comment:
tiny nit but could we add the space between `mode.x` and the `=`? I think we're pretty consistent about that spacing in the docs so far

Reply:
Not a problem, just fixed it!